### DISCUSSION TITLE
Took the leap..

### DISCUSSION THREAD

**Discussion by letsCode31** [Cyfrin/foundry-full-course-cu#2395] [General]
Hello everyone. 

Firstly, thanks Patrick and team for making something that even beginners like me can follow . /\

I am a sales & marketing professional, looking to re-invent myself and took the plunge of learning blockchain coding. I have some prior knowledge about coding, from my engineering days.. but that was 15 years back.. pheww!

Glad to be here. looking forward to learn and grow, in this field, with the awesome team and community we have here..

**Comment by EngrPips** [comment]
Welcome aboard. We are glad to have you. We are always here to help and support you, and we wish you the best in your journey.
